src/subscription/UpgradeSubscriptionPage.ts

```diff
@@ -207,7 +207,7 @@ function confirmFreeSubscription(): Promise<boolean> {
 			dialog.close()
 			setTimeout(() => resolve(confirmed), DefaultAnimationTime)
 		}
-
+		const isFormValid = stream.lift((_oneAccountValue, _privateUseValue) => _oneAccountValue && _privateUseValue, oneAccountValue, privateUseValue)
```

Comment:
I think I would just make a method instead of a new stream, we don't need to listen to it

Reply:
Fair point, changed.